files/localSettings.xml

```diff
@@ -0,0 +1,4 @@
+<settings xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/settings-1.0.0.xsd">
+    <localRepository>E:\Testowanie\repos\test-automation-template\temp\deps</localRepository>
```

Comment:
zapomniałeś zmienić ścieżke

Reply:
wywaliłem to localSettings.xml zamiast tego w readme dałem komendę `mvn clean install -Dmaven.repo.local=temp\deps`